src/main/java/ballmerpeak/stargate/proto/ProtoIO.java

```diff
@@ -23,6 +23,7 @@
 	
 	public ProtoIO(Game game) {
 		tileInfo.put(DrawableIndex.FLOOR_WITH_ZPM, "Floor <ZPM>");
+		tileInfo.put(DrawableIndex.FLOOR_WITH_ZPM2, "Floor <ZPM>");
```

Comment:
fail of multiple ZPMs
(and because the proto uses drawableIndex information, isnt dat a gui stuff m8?